modules/460-log-shipper/crds/cluster-log-destination.yaml

```diff
@@ -660,6 +660,9 @@ spec:
                       app_info: '{{ app }}'
                       array_member: '{{ array[0] }}'
                       symbol_escating_value: '{{ pay\.day }}'
+                  x-kubernetes-validations:
+                    - message: parsed_data cannot be used as a key in extraLabels, it's reserved
```

Comment:
I think we have to exclude all `system` words here
https://deckhouse.io/documentation/v1/modules/460-log-shipper/cr.html#clusterlogdestination-v1alpha1-spec-extralabels